examples/maturin-starter/.template/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+authors = ["PyO3 Authors"]
+name = "maturin-starter"
```

Comment:
Both the crate name and the library name appear unchanged after running cargo-generate. Can this handled by the template functionality of cargo-generate?

Reply:
Good point, will fix.

Reply:
Hmm, actually, the crate name is trivial, but the library name is really hard to change because it's _everywhere_ across the whole example. I don't have a good solution for that for now, so I'll only change the crate name.